macros/CSS_Ref.ejs

```diff
@@ -13,12 +13,12 @@
        defaults to ['standard'])
 */
 
-var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/properties.json";
-var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/at-rules.json";
-var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/selectors.json";
-var typesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/types.json";
-var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/syntaxes.json";
-var unitsUrl = "https://raw.githubusercontent.com/mdn/data/mdn/css/units.json";
+var propertiesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/properties.json";
+var atRulesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/at-rules.json";
+var selectorsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/selectors.json";
+var typesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/types.json";
+var syntaxesUrl = "https://raw.githubusercontent.com/mdn/data/master/css/syntaxes.json";
+var unitsUrl = "https://raw.githubusercontent.com/mdn/data/master/css/units.json";
```

Comment:
Tested on [Web/CSS/Reference](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference).  Renders, with changes:

Removed links:

* [annotation()](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-alternates#annotation())
* ``cubic-bezier()``
* ``format()`` and ``format() (@font-face)``
* ``local()``
* ``ornaments()``
* ``rect()``
* ``repeat()``
* ``steps()``
* ``styleset()``
* ``@stylistic``
* ``stylistic()``
* ``swash()``
* ``:unresolved``

Added Links:

* [&lt;an-plus-b&gt;](https://developer.mozilla.org/docs/Web/CSS/an-plus-b)